runtime/runtime.go

```diff
@@ -45,6 +45,7 @@ type RunConfig struct {
 	MAC          string
 	Cmd          string
 	Persist      bool
+	Volumes      []string
```

Comment:
Same thing for grouping/sorting parameters as i wrote above.

Reply:
Done.